.circleci/config.yml

```diff
@@ -0,0 +1,109 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
+    - checkout
+
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./architect
+        ./architect version
+
+    - run: ./architect build
+
+    - persist_to_workspace:
+        root: .
+        paths:
+        - ./route53-manager
+
+  syncRecordsets:
+    environment:
+      SOURCE_REGION: "cn-north-1"
+      TARGET_REGION: "eu-central-1"
+    machine: true
+    steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run: ./route53-manager
+
+
+workflows:
+  version: 2
+
+  build:
+    jobs:
+    - build
+
+  sync:
+    triggers:
+      - schedule:
+          cron: "0 * * * *"
```

Comment:
As CircleCI uses POSIX cron syntax ([source](https://circleci.com/docs/2.0/configuration-reference/#schedule)), this could be written like this:

```yaml
cron: "0,5,10,15,20,25,30,35,40,45,50,55 * * * *"
filters:
  branches:
    only: master
```

This may be a matter of taste, but I like the more concise version. No strong opinion though. :)

Reply:
Nice! Much better this way :)